packages/rum-core/src/domain/requestCollection.ts

```diff
@@ -159,7 +159,7 @@ function getNextRequestIndex() {
   return result
 }
 
-function waitForResponseToFinish(context: RumFetchCompleteContext, callback: (duration: Duration) => void) {
+function waitForResponseToFinish(context: RumFetchResolveContext, callback: (duration: Duration) => void) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** `waitForResponseToComplete` would avoid to use two terms for the same concept‏